templates/Default/admin/Default/1.6/universe_create_locations.php

```diff
@@ -0,0 +1,43 @@
+Working on Galaxy : <?php echo $Galaxy->getName(); ?> (<?php echo $Galaxy->getGalaxyID() ?>)<br /><br />
+
+<?php
+foreach ($LocTypes as $category => $LocIDs) { ?>
+		<a href="javascript:;" onclick="window.location.hash='<?php echo $category; ?>'">Jump to <?php echo $category; ?></a><br /><?php
+} ?>
+
+<br />
+Click a category heading to toggle its display.
+
+<?php echo $Form; ?>
```

Comment:
Can you change this to
```php
$template->assign('CreateLocationsFormHREF', SmrSession::getNewHREF($container));
...
<form method="POST" action="<?php echo $CreateLocationsFormHREF; ?>">
```
so we can get the full benefits of html handling in the template and get rid of another use of `create_echo_form`
(naming of the vars are up to you, I just tried to choose one that made sense)